LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 
+Copyright (c) 2024
```

Comment:
This should be reverted. It's not our copyright.

Reply:
I see. I fixed it now. https://github.com/Screenly/Playground/pull/162/commits/f456ba38d5ae82290e3182acf378b6b949ca3595